api/k8s/v1/k8s.proto

```diff
@@ -509,6 +517,35 @@ message DeleteCronJobRequest {
 message DeleteCronJobResponse {
 }
 
+message ConfigMap {
+  option (clutch.api.v1.id).patterns = {
+    type_url : "clutch.k8s.v1.ConfigMap",
+    pattern : "{cluster}/{namespace}/{name}"
+  };
+
+  string cluster = 1;
+  string namespace = 2;
+  string name = 3;
+
+  map<string, string> data = 4;
```

Comment:
What data is intended to be put in this field?

Reply:
removed 👍